datasetinsights/commands/download.py

```diff
@@ -42,20 +43,6 @@ def convert(self, value, param, ctx):
     help="Download datasets to localhost from known locations.",
     context_settings=const.CONTEXT_SETTINGS,
 )
-@click.option(
-    "-n",
-    "--name",
-    type=click.STRING,
-    required=True,
-    help="The dataset registry name.",
-)
-@click.option(
-    "-d",
-    "--data-root",
-    type=click.Path(exists=True, file_okay=False, writable=True),
-    default=const.DEFAULT_DATA_ROOT,
-    help="Root directory on localhost where datasets should be downloaded.",
-)
 @click.option(
```

Comment:
I think it might be better that we make source-uri required parameter. Otherwise, the downloader usually don't know where it should download from.